.pipelines/templates/install-k8s-extension.yaml

```diff
@@ -0,0 +1,7 @@
+steps: 
+  - bash: |
+      az extension remove --name k8s-extension
+      git clone https://github.com/Azure/azure-arc-kubernetes-preview
+      az extension add --source ./extensions/k8s_extension-${k8s_extension.version}-py2.py3-none-any.whl --yes
```

Comment:
is this path correct for the source file? Don't you have to be in the azure-arc-kubernetes-preview directory first?

Reply:
Good catch, I also need to add `$(System.DefaultWorkingDirectory)/azure-arc-kubernetes-preview` when I run `git clone`